backend/routes/userEndpoints.js

```diff
@@ -42,6 +42,37 @@ userEndpoints.route("/users/add").post(function (req, response) {
   });
 });
 
+// Endpoint to login as a particular user.
+userEndpoints.route("/login").post(function (req, response) {
+  let db_connect = databaseConnection.getDb();
+  let myobj = {
+    _id: req.body.email,
+  };
+
+  db_connect.collection("records").findOne(myobj, function (err, user) {
+    if (err) throw err;
+    if (!user) {
+      throw {error: 'User Not Found!'};
+    } else {
+      if (checkPassword(user.password, req.body.password)) {
```

Comment:
Shouldn't this use bcrypt to check password?
    bcrypt.compare(password, user.password).then(isMatch => {
      if (isMatch) {
        // User matched
        // Create JWT Payload
        const payload = {